### PR TITLE
[DOCS-7261] Ansible install for ACS 7.4

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

File: content-services/latest/install/ansible.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Install with Ansible
 ---
 
-This page describes how to install Content Services using an [Ansible](https://www.ansible.com){:target="_blank"} playbook. Ansible is an open-source software provisioning, configuration management, and application installation tool that enables infrastructure as code. Alfresco provides an Ansible playbook capable of installing Content Services (Enterprise Edition) version 7.3.
+This page describes how to install Content Services using an [Ansible](https://www.ansible.com){:target="_blank"} playbook. Ansible is an open-source software provisioning, configuration management, and application installation tool that enables infrastructure as code. Alfresco provides an Ansible playbook capable of installing Content Services (Enterprise Edition) version .
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:5]: It's either missign the version number or the word "version" should be removed completely? I'd rather we remove any mention of version given the playbook can install several versions.

File: content-services/latest/install/ansible.md [hunk 2]
```diff
@@ -85,73 +84,93 @@ If you're using the Enterprise edition of Content Services, then you need creden
...
 
-1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.2.1/alfresco-ansible-deployment-2.2.1.zip){:target="_blank"}.
+1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.3.0/alfresco-ansible-deployment-2.3.0.zip){:target="_blank"}.
 
    If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the control node.
 
    ```bash
    scp  alfresco-ansible-deployment-<version>.zip user@controlnode:
-   scp  ~/.ssh/ansible_rsa user@controlnode:.ssh
+   scp  -i ~/.ssh/ansible_rsa user@controlnode:.ssh
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:155]: We have an error in our doc here.
 Please use instead:
```suggestion
  ssh-copy-id -i ~/.ssh/ansible_rsa user@controlnode
```

File: content-services/latest/install/ansible.md [hunk 3]
```diff
@@ -242,7 +261,7 @@ The diagram below shows the result of a local installation.
 
 ![acs-localhost]({% link content-services/images/acs-localhost.png %})
 
-To install Content Services 7.3 (Enterprise) on your local machine, navigate to the folder where you extracted the ZIP, and run the playbook as the current user (the playbook will escalate privileges when required):
+To install Content Services  (Enterprise) on your local machine, navigate to the folder where you extracted the ZIP, and run the playbook as the current user (the playbook will escalate privileges when required):
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:264]:
```suggestion
To install Content Services (Enterprise) on your local machine, navigate to the folder where you extracted the ZIP, and run the playbook as the current user (the playbook will escalate privileges when required):
```

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: content-services/latest/install/ansible.md (hunk 1 above)
Comment (anxumalo) [reply content-services/latest/install/ansible.md:5]: I agree - removing the version number also reduces the maintenance we have to do for each new release.

Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: I removed the need to update each version by adding your suggestion Alex.

File: content-services/latest/install/ansible.md (hunk 2 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: I added the correction to a new commit. Thank you.

File: content-services/latest/install/ansible.md [hunk 4]
```diff
@@ -527,7 +557,7 @@ If you have a valid license, place your `*.lic` file in the `configuration_files
 
 ### Alfresco/Solr authentication
 
-As of Content Services 7.2 and/or Search services 2.0.3, the repository <--> solr communication requires authentication. The playbook will set up that authentication scheme using the new `secret` method. This methods need to be passed a shared secret. In order to do so use the variable below:
+As of Content Services 7.2 and/or Search services 2.0.3, the repository <--> Solr communication requires authentication. The playbook will set up that authentication scheme using the new `secret` method. This methods need to be passed a shared secret. In order to do so use the variable below:
```
Comment (anxumalo) [inline content-services/latest/install/ansible.md:560]: I assume the `<-->` comment marker is not needed, so I'll delete it.

**Review by anxumalo — APPROVED** [review]
LGTM

**Review by gionn — APPROVED** [review]

**Pull request merged**